cyipopt/tests/unit/test_scipy_optional.py

```diff
@@ -14,6 +14,10 @@
 
 import cyipopt
 
+# Hard-code rather than importing from scipy.optimize._minimize in a try/except
+MINIMIZE_METHODS = ['nelder-mead', 'powell', 'cg', 'bfgs', 'newton-cg',
+                    'l-bfgs-b', 'tnc', 'cobyla', 'slsqp', 'trust-constr',
+                    'dogleg', 'trust-ncg', 'trust-exact', 'trust-krylov']
```

Comment:
Would you prefer:
```suggestion
try:
    from scipy.optimize._minimize import MINIMIZE_METHODS
except ImportError:
    pass
```
?

Reply:
We'll have to maintain this hard coded list if things change in scipy, so I think I'd prefer the try clause. Is there a reason you hardcoded?

Reply:
Only to avoid importing from a private module. But I don't think that's much of a concern to me anymore.